cmd/powd/main.go

```diff
@@ -301,6 +308,8 @@ func setupFlags() error {
 	pflag.String("maxminddbfolder", ".", "Path of the folder containing GeoLite2-City.mmdb")
 	pflag.String("mongouri", "", "Mongo URI to connect to MongoDB database. (Optional: if empty, will use Badger)")
 	pflag.String("mongodb", "", "Mongo database name. (if --mongouri is used, is mandatory")
+	pflag.String("ffsminerselector", "sr2", "Miner selector to be used by FFS: 'sr2', 'reputation'")
```

Comment:
`MinerSelector` can be:
- `sr2`: the default now. Does what was commented in PR description.
- `reputation`: The previous miner selector that uses the reputation index, and still directly targets peter's miners as trusted.